references.bib

```diff
@@ -84,18 +98,14 @@ @Website{pca-visually-explained
 @Manual{Prabhakaran2016,
   title = {The Complete ggplot2 Tutorial},
   author = {Selva Prabhakaran},
-  year = {2016-2017},
+  year = {2016},
   url = {http://r-statistics.co/Complete-Ggplot2-Tutorial-Part1-With-R-Code.html},
 }
 
-@Article{Micol2017,
-   Author="Micol, J. B.  and Pastore, A.  and Inoue, D.  and Duployez, N.  and Kim, E.  and Lee, S. C.  and Durham, B. H.  and Chung, Y. R.  and Cho, H.  and Zhang, X. J.  and Yoshimi, A.  and Krivtsov, A.  and Koche, R.  and Solary, E.  and Sinha, A.  and Preudhomme, C.  and Abdel-Wahab, O. ",
-   Title="{{A}{S}{X}{L}2 is essential for haematopoiesis and acts as a haploinsufficient tumour suppressor in leukemia}",
-   Journal="Nat Commun",
-   Year="2017",
-   Volume="8",
-   Pages="15429",
-   Month="05"
+@Website{prcomp-function,
```

Comment:
Looks like we have two of these for `prcomp`? I don't think we need either of them.